webpack/scenes/Subscriptions/SubscriptionConstants.js

```diff
@@ -24,4 +24,4 @@ export const BLOCKING_FOREMAN_TASK_TYPES = [
 ];
 
 export const MANIFEST_TASKS_BULK_SEARCH_ID = 'activeManifestTasksSearch';
-export const BULK_TASK_SEARCH_INTERVAL = 10000;
+export const BULK_TASK_SEARCH_INTERVAL = 4000;
```

Comment:
10 seconds interval doesn't catch the `DeleteManifest` task

Reply:
Hmm, thoughts on this @tstrachota?  I suppose lowering the interval would be okay with your changes to make the polling stop?

Reply:
I'm not sure I understand. What is the reason for lowering the interval?

Reply:
after a click on `Delete Manifest` button, a task is triggered. with 10 seconds interval this task mostly won't be caught, so the modal won't close, and the subscriptions list won't update.

Reply:
I see. How about triggering one additional task search request on a successful response from any manifest action? The reducer doesn't care when and updates the running tasks as long as the action has `MANIFEST_TASKS_BULK_SEARCH_ID`.